src/Ganeti/HTools/CLI.hs

```diff
@@ -259,6 +261,7 @@ defaultOptions  = Options
   , optNodeSim     = []
   , optNodeTags    = Nothing
   , optSkipNonRedundant = False
+  , optStaticKvmNodeMemory = 4096
```

Comment:
Yes indeed. This provides backward compatibility - i.e. no need to add the flag to every execution of hail, hbal etc. which could be problematic if you have a mixed fleet of 2.16 (i.e. you'd need to detect which cluster was upgraded and set the flag accordingly).